frontend/src/pages/team/Devices/dialogs/TeamDeviceCreateDialog.vue

```diff
@@ -126,15 +148,13 @@ export default {
         }
     },
     async mounted () {
-        if (this.features.billing && // billing enabled
-                this.team.type.properties.billing?.deviceCost > 0) {
-            // Get the total device count for the team so that we can check allocation
-            // By passing 0 in as the limit, it will just return the `count`
-            // and not retrieve any actual devices
-            const teamData = await teamApi.getTeamDevices(this.team.id, null, 0)
-            this.totalDeviceCount = teamData.count
-        }
         this.loadApplications()
+        if (this.features.billing && !this.team.billing?.unmanaged) {
+            try {
+                this.subscription = await billingApi.getSubscriptionInfo(this.team.id)
+            } catch (err) {
```

Comment:
why nothing in the catch here? Will suppress an error that we should be made aware of, no?

Reply:
Because if the team does not have billing setup, this end point returns a 404. This is consistent with all other places we retrieve the subscriotuin